nextflow.config

```diff
@@ -21,8 +21,9 @@ params {
     diginorm_k                  = 20
     orf_caller                  = 'prodigal'
     prodigal_trainingfile       = null
-    eukulele_db                 = null
+    eukulele_db                 = 'phylodb'
     skip_eukulele               = false
+    eukulele_dbpath             = null
```

Comment:
You got me wrong here, I think. I think we should have a default for the `eukulele_dbpath`, because otherwise the large database will just be put into a work directory. We could set the default for the path to e.g. `eukulele_db`.
I think having `phylodb` as default for the `eukulele_db` makes sense as prokaryotes will likely be the most common usecase. One could argue for not having a default at all, to ensure that the user actively chooses a database after considering the available options. In the end, I'm happy with having a `phylodb` default.

Reply:
Should I do something like this:
`eukulele_dbpath = '../phylodb/' ` as default?
so the database will be downloaded out of the working directory?

Reply:
Not the two periods, just `phylodb`. Then, a directory with this name will be created and files downloaded there. A bit clearer for the end user wanting to reuse the files.